src/main/java/org/openrewrite/hibernate/hibernate60/MigrateHypersistenceUtils61Types.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.hibernate.hibernate60;
```

Comment:
Please add a `package-info.java` file to this new package like the following one: https://github.com/openrewrite/rewrite/blob/2d4492a4d296bc692d501c3d7c44f4d02abf41dc/rewrite-java/src/main/java/org/openrewrite/java/package-info.java. This will resolve some of the nullability warnings in IDEA.